src/Administration/Resources/app/administration/src/app/component/form/sw-colorpicker/sw-colorpicker.scss

```diff
@@ -298,9 +288,6 @@
             border: 1px solid $color-steam-cloud;
             border-radius: $border-radius-default;
             background: #fff;
-        }
-
-        &::-moz-range-thumb:hover {
             cursor: pointer;
         }
```

Comment:
Please let this css selector still exist with its content and add an deprecation tag E.g. `/* @deprecated tag:v6.3.0 */`. Duplicate the css cursor prop and put it in the right selector as you did before.